bin/network-monitor/src/deploy/counter.rs

```diff
@@ -14,12 +14,24 @@ use miden_objects::account::{
     AccountStorageMode,
     AccountType,
     StorageSlot,
+    StorageSlotName,
 };
+use miden_objects::utils::sync::LazyLock;
 use miden_objects::{Felt, FieldElement, Word};
 use tracing::instrument;
 
 use crate::COMPONENT;
 
+static OWNER_SLOT_NAME: LazyLock<StorageSlotName> = LazyLock::new(|| {
+    StorageSlotName::new("miden::counter_contract::owner")
+        .expect("storage slot name should be valid")
+});
+
+static COUNTER_SLOT_NAME: LazyLock<StorageSlotName> = LazyLock::new(|| {
+    StorageSlotName::new("miden::counter_contract::counter")
+        .expect("storage slot name should be valid")
+});
```

Comment:
nit (feel free to ignore): I'd maybe make the names a bit more "narrow" - i.e., `miden::monitor::counter_contract::counter` and `miden::monitor::counter_contract::owner`.